README.rst

```diff
@@ -80,7 +80,7 @@ Quick Example
         await ctx.send(f"Hello {name}!")
         
     @bot.user_command(name="Say Hello")
-    async def hi(ctx, user : discord.User):
+    async def hi(ctx, user : discord.Member):
```

Comment:
Remove the whitespace between "user" and ":". See [this](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements)

Reply:
Like this 

user:discord.Member?

Reply:
No, Follow the PEP-8 code styling conventions.
```
user: discord.Member
```

Reply:
Alr ty, I just commited the change